packages/flutter_rx_bloc/.gitignore

```diff
@@ -0,0 +1 @@
+*.mocks.dart
```

Comment:
All code should reside in the repo, even the generated one. We do this because we want to provide the best dev experience otherwise, the package won't compile until the code generator is not being executed.

Reply:
Done.